javascript/code-challenges/challenges-14.test.js

```diff
@@ -0,0 +1,308 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named screenForNames that takes in an array of strings and uses Regex to create a new array of only those strings that match the following rules:
+
+* the name must begin with Mr., Mrs., Ms., Dr. followed by a space
+* the name must contain only letter characters (white spaces are ok)
+
+------------------------------------------------------------------------------------------------ */
+
+const screenForNames = (arr) => {
+  let array = [];
+  let regex = /^(Dr|Mr|Ms|Mrs)\.\s\w+/g;
+  arr.map(element => {
+    if(regex.test(element)){
+      array.push(element);
+    }
+  });
+  return array;
+  // Solution code here...
+}
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named toTitleCase that takes in an array of strings and returns an array of strings with the first character in upper case and the rest as is.
+
+For example, ['apple', 'banana', 'MacGyver'] returns ['Apple', 'Banana', 'MacGyver'].
+------------------------------------------------------------------------------------------------ */
+
+const toTitleCase = (arr) => {
+  let marco = arr.map(element => element.split(''));
+
+  let polo = marco.map(element => {
+    let travler = element[0].toUpperCase();
+    for(let i=1; i<element.length; i++){
+      travler = travler + element[i];
+    }
+    return travler;
+  });
+
+
+  return polo;
```

Comment:
Really good job on all the challenges!

Now that you have found solution, maybe we can think about how we can simplify a bit.

You are mapping/iterating through the elements in the input arr. Feedback here would be to remove the two additional loops and see if you can upperCase item at index 0 in the input array after the callback. 👍